src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -807,8 +807,8 @@ def max_timestamp(aggregate_filter, org_id, use_case_id, alias=None):
     return timestamp_column_snql("maxIf", aggregate_filter, org_id, use_case_id, alias)
 
 
-def total_count(aggregate_filter: Function) -> Function:
-    return Function("sumIf", [Column("value"), aggregate_filter])
+def total_count(aggregate_filter: Function, alias=None) -> Function:
```

Comment:
Could you please merge master? I merged a bunch of typing changes for this file. You can then match `alias` type to the other ones.